app/models/search_tab.rb

```diff
@@ -25,7 +26,7 @@ def self.add_category_facets(search, category)
   end
 
   def self.valid_category_facets
-    Search.new.facet_values("category").keys
+    Search.new.facet_values('category').keys
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.